src/components/FlightFlagger/FlightFlaggerTable.tsx

```diff
@@ -36,16 +46,51 @@ export const FlightFlaggerTable = ({flights, ...others}: IFlightFlaggerTable) =>
         <StyledTableHeaderCell>Flight</StyledTableHeaderCell>
         { hasHighlightedFlights && <StyledTableHeaderCell>Pax info</StyledTableHeaderCell> }
         <StyledTableHeaderCell align="center">Origin</StyledTableHeaderCell>
-        <StyledTableHeaderCell align="center">Country</StyledTableHeaderCell>
+        <StyledTableHeaderCell align="center">
+          Country
+          <StyledTooltip title="Country of origin">
+            <IconButton>
+              <InfoIcon />
+            </IconButton>
+          </StyledTooltip>
+        </StyledTableHeaderCell>
         <StyledTableHeaderCell align="center">Gate/Stand</StyledTableHeaderCell>
         <StyledTableHeaderCell align="center">Status</StyledTableHeaderCell>
         <StyledTableHeaderCell align="center">Scheduled</StyledTableHeaderCell>
-        <StyledTableHeaderCell align="center">Expected</StyledTableHeaderCell>
+        <StyledTableHeaderCell align="center">
+          Expected
+          <StyledTooltip title="Expected arrival">
+            <IconButton>
+              <InfoIcon />
+            </IconButton>
+          </StyledTooltip>
+        </StyledTableHeaderCell>
         <StyledTableHeaderCell align="center">Exp PCP</StyledTableHeaderCell>
         <StyledTableHeaderCell align="center">Exp PCP Pax</StyledTableHeaderCell>
-        <StyledTableHeaderCell align="center">e-Gate</StyledTableHeaderCell>
-        <StyledTableHeaderCell align="center">EEA</StyledTableHeaderCell>
-        <StyledTableHeaderCell align="center">Non-EEA</StyledTableHeaderCell>
+        <StyledTableHeaderCell align="center">
+          e-Gate
+          <StyledTooltip title="Expeced total pax">
```

Comment:
Minor typo here and below 'Expeced'

Reply:
Oh actually I guess this should say 'Expected e-Gate pax', and below EEA & Non-EEA - though I also just realised this component isn't going to be used yet and we'll need to pass the splits dynamically so I guess it's fine to leave for now

Reply:
I'll fix the typo! But yeah this component will need some work before we can use it.